create_blog_post_using_azure_openai_and_publish_in_medium/README.md

```diff
@@ -0,0 +1,34 @@
+# Generate a blog article using Azure OpenAI and publish on Medium
+
+## Use Case
+Generates content in markdown format for a blog article for a given topic using a text completion model from Azure OpenAI and publishes the content as a blog post on Medium.
+
+## Prerequisites
+* Azure OpenAI service
+* Medium account
+
+## Setting up Azure OpenAI service
+1. Follow the instructions given in [Azure Documentation](https://learn.microsoft.com/en-us/azure/cognitive-services/openai/how-to/create-resource?pivots=web-portal) to create an Azure OpenAI resource and deploy a text completion model.
+2. In the Azure Portal, go to **Keys and Endpoint** in the **Resource Management** section of the resource and obtain the service URL (**Endpoint**) and API key (either **KEY1** or **KEY2**).
+3. Go to **Deployments** in the **Resource Management** section and obtain the name of the text completion model which was deployed in step 1.
+
+## Setting up Medium account
+1. Sign up or sign in to [Medium](https://medium.com/)
```

Comment:
Shall we change this to `Sign up and log into`?

Reply:
Addressed in [cd7ff8d](https://github.com/ballerina-guides/ai-samples/pull/62/commits/cd7ff8d2ed170a118ee35fd6c1a6ab8d42b2fab8)